src/plugins/expressions/common/executor/executor.ts

```diff
@@ -40,6 +40,7 @@ export interface ExpressionExecOptions {
 }
 
 export class TypesRegistry implements IRegistry<ExpressionType> {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
why are we providing `any` here ? seems we should just avoid providing generic and rely on defaults.